src/test/java/api/LoanRulesAPITests.java

```diff
@@ -34,14 +26,14 @@ public void canGet() throws Exception {
   public void canPutAndGet() throws Exception {
     String rule = "priority: t, s, c, b, a, m, g\nfallback-policy: no-circulation\n";
 
-    Response response = put(rule);
-    assertThat(response.getStatusCode(), is(204));
+    loanRulesFixture.updateLoanRules(rule);
+
```

Comment:
Why status code check has been removed here?

Reply:
Thank you reviewing.

The loan rules fixture performs the [same check](https://github.com/folio-org/mod-circulation/pull/176/files#diff-779dc7083f3bd960aab8805a0fea700fR51) and so this is how redundant.